.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,110 @@
+---
+name: Feature request
+about: Suggest a new feature for this project
+title: "[NEW FEATURE REQUEST]"
+labels: Feature
+assignees: ''
+
+---
+
+## General information
+
+- Applicant:
+- Developer:
+- User group referent:
+- Developments referent:
+
+## Users and administrators needs
+
+### Graphical identity / Logo
+
+_Illustrations if applicable._
+
+### Description of users and administrators feature requirements
+
+_Describe the new feature_
+
+#### General requirements
+
+
+#### Specific requirements
+
+Necessary features:
+### Illustrations of the features and the requirements
+
+_Schemes, screen captures, or any other illustrations._
+
+
+### Graphical User Interface (GUI)
+
+
+## Technical analysis and specifications
+
+### Feasibility study
+
+Feasibility, options, alternatives and technical specificities
+
+
+### Risks and constraints
+
+
+### Mutualisation
+_Mutualisation and possible impacts with other projects_
+
+
+## Specification and offers ( Developer )
+Not shared on Github.
```

Comment:
Maybe just set 
- [ ] specification exist
- [ ] offers handed in 
- [ ] offer accepted
or we completely take this out here.